### DISCUSSION TITLE
Meta not showing in ssr :true

### DISCUSSION THREAD

**Discussion by AO-IO** [nuxt/nuxt#23066] [Questions]
I'm using nuxt 3 . And in my index page I've set A `useSeoMeta` composable.

```JAVBASCRIPT

<script setup>

useSeoMeta({
   title: 'title',
   ogTitle: 'title',
   description: 'lorem lorem',
   ogDescription: 'lorem lorem.',
   ogImage: 'cover-image.png',
   twitterCard: 'summary_large_image',
})


</script>
<template>
   <div></div>
</template>

``` 

This works as expected but its not rendering the tags in the page when inspecting the page code.

I have tried to use `useServerSeoMeta` but its not working . No tags are rendering or showing in the browser tab .

The `ssr` option in the config is enabled .

Any idea what am i doing wrong here?

**Comment by logotip4ik** [comment]
Would you please provide reproduction ?
It works for [fresh project](https://stackblitz.com/edit/nuxt-starter-tuve1w?file=app.vue). Open a dev tools and pick preview element
